test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -239,52 +239,58 @@ std::string RandomCodeBase::rndByteSequence(int _length, SizeStrictness _sizeTyp
 	return hash;
 }
 
+bool isOpcodeDefined(uint8_t _opcode)
+{
+	eth::Instruction inst = (eth::Instruction) _opcode;
+	eth::InstructionInfo info = eth::instructionInfo(inst);
+	if (info.name.find("INVALID_INSTRUCTION") != std::string::npos || info.name.empty()
```

Comment:
`if (...) return true; else return false;` is not very smart. Just return check condition value.